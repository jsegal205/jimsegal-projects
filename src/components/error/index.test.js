import React from "react";
import { render, cleanup } from "@testing-library/react";
import Error from "./index";

afterEach(cleanup);

describe("Error Component", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Error />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("with prop renders correctly", () => {
    const { asFragment } = render(<Error componentName="ERROR" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
