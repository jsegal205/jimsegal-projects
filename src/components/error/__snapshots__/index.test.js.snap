// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Component renders correctly 1`] = `
<DocumentFragment>
  <section>
    <h3>
      Whoops! There was a problem loading data.
    </h3>
    <p>
      Please reload browser to try again in a little bit.
    </p>
  </section>
</DocumentFragment>
`;

exports[`Error Component with prop renders correctly 1`] = `
<DocumentFragment>
  <section>
    <h3>
      Whoops! There was a problem loading ERROR data.
    </h3>
    <p>
      Please reload browser to try again in a little bit.
    </p>
  </section>
</DocumentFragment>
`;
